### DISCUSSION TITLE
Saving info

### DISCUSSION THREAD

**Discussion by gringojack** [mytourbook/mytourbook#1023] [General]
Hi
Is there any possibility to save daily measures or values (heartrate at night, weigh, blood pressure, etc...)

**Comment by wolfgang-ch** [comment]
Anything which can be saved in MT must be contained in a tour or is related to a tour, e.g. tour marker. 

A manual tour can be created and saved but may not yet contain all values which you want.

**Reply by gringojack** [reply]
Thank you for these pieces of info

**Comment by JoerniCarboni** [comment]
Hi there, I'm new at tourbook and struggle how I can save changes I made in a tour. Is there anywhere a button I haven't seen?
I found the option to save "imported tours", but is with changes I made in a tour
![Screenshot 2024-02-20 113058](https://github.com/mytourbook/mytourbook/assets/160596890/32e34bcb-bba5-4618-8748-3996ad749582)

**Reply by FJBDev** [reply]
@JoerniCarboni 

Welcome to MyTourbook!

As a new user, I would recommend to read this well made "[First Steps](https://mytourbook.sourceforge.io/mytourbook/index.php/first-steps)" documentation, it should help you to save changed in your tour.

I hope this helps!

**Reply by FJBDev** [reply]
@JoerniCarboni 

Also, if I understand correctly, this is the button to save changes made in the Tour Data Editor View

![image](https://github.com/mytourbook/mytourbook/assets/25235665/a95c57ff-2ab5-4fa7-abe5-700f935ab12b)

**Reply by FJBDev** [reply]
And one last thing to note: `Ctrl+S` should work! 
See the shortcuts [here](https://mytourbook.sourceforge.io/mytourbook/images/mytb/CheatSheets/MT-CheatSheet-23.10.pdf)
